Tests/IndexStoreDBTests/IndexTests.swift

```diff
@@ -317,6 +309,8 @@ final class IndexTests: XCTestCase {
       enableOutOfDateFileWatching: true,
       listenToUnitEvents: true
     )
+    // Wait a bit for the notification to come through.
+    sleep(1)
```

Comment:
I assume there's nothing better to wait on here :(?

Reply:
yeah, this is junky, I changed it to wait on an XCTest expectation instead.